manifests/install.pp

```diff
@@ -0,0 +1,13 @@
+# install.pp
+# Installs clamav package
+#
+
+class clamav::install {
+
+  package { 'clamav':
+    ensure  => installed,
+    name    => $clamav_package,
```

Comment:
This variable needs to be fully qualified or have a local copy like the other classes...